### PR TITLE
[SYMFONY43] Move to 4.4 changes in GetResponseForExceptionEvent

### PR DESCRIPTION
**Pull request by gander**
Closes https://github.com/rectorphp/rector-symfony/issues/346

### CODE REVIEW

**Comment by gander** [comment]
Oops. I ran the test at home, on 8.2, and it was ok.

**Comment by TomasVotruba** [comment]
The `Symfony43Test` needs an update

**Comment by gander** [comment]
Yes, I read in the Github Actions log. I just forgot about that.

**Comment by gander** [comment]
I ran the tests locally but had an error whose source I don't understand.

**Comment by gander** [comment]
I do not know what's going on. After all, it was in 4.3 and it is in 4.4
```
There was 1 failure:

1) Rector\Symfony\Tests\Set\Symfony44\Symfony44Test::test with data set #0
Failed asserting that string matches format description.
--- Expected
+++ Actual
@@ @@
 {
     public function someMethod(\Symfony\Component\HttpKernel\Event\ExceptionEvent $exceptionEvent)
     {
-        $exception = $exceptionEvent->getThrowable();
-        $exceptionEvent->setThrowable($exception);
+        $exception = $exceptionEvent->getException();
+        $exceptionEvent->setException($exception);
     }
 }
```

File: tests/Set/Symfony44/Fixture/event_fixture.php.inc [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?php
+
+class SomeEventListener
```
[the remark this answers is not in the document]
Comment (gander) [reply tests/Set/Symfony44/Fixture/event_fixture.php.inc:3]: No, that's not it. The test loads these files correctly. But replace doesn't seem to happen.

[the remark this answers is not in the document]
Comment (gander) [reply tests/Set/Symfony44/Fixture/event_fixture.php.inc:3]: Oh, I think I know what the error is.

[the remark this answers is not in the document]
Comment (gander) [reply tests/Set/Symfony44/Fixture/event_fixture.php.inc:3]: Yes, I had it in front of my eyes all the time. Wrong class.

File: tests/Set/Symfony43/Fixture/event_fixture.php.inc [hunk 2]
```diff
@@ -17,8 +15,6 @@ class SomeEventListener
 {
     public function someMethod(\Symfony\Component\HttpKernel\Event\ExceptionEvent $exceptionEvent)
     {
-        $exception = $exceptionEvent->getThrowable();
-        $exceptionEvent->setThrowable($exception);
     }
 }
```
Comment (TomasVotruba) [inline tests/Set/Symfony43/Fixture/event_fixture.php.inc:20]: This file can be removed completelly, as it does not show any change now.

**Comment by gander** [comment]
Guess I'll have to do a Squash Merge to clean up here

File: tests/Set/Symfony43/Fixture/event_fixture.php.inc (hunk 2 above)
Comment (gander) [reply tests/Set/Symfony43/Fixture/event_fixture.php.inc:20]: No, the class is changed.

Comment (TomasVotruba) [reply tests/Set/Symfony43/Fixture/event_fixture.php.inc:20]: Ah, I see :+1:

**Comment by gander** [comment]
It was my mistake. For the new tests, I used a changed class from other tests.

**Comment by TomasVotruba** [comment]
Thank you :clap:

**Pull request merged**